src/LogarithmicNumbers.jl

```diff
@@ -605,6 +605,10 @@ function Base.exp(x::Logarithmic)
     x.signbit ? inv(exp(x.abs)) : exp(x.abs)
 end
 
+function Base.sqrt(x::AnyLogarithmic)
+    uexp(x.log/2)
```

Comment:
please change to `uexp(x.log / oftype(x.log, 2))` in order to preserve the type of `x.log`